### PR TITLE
Fix TokTok.jl

### PR DESCRIPTION
**Pull request by Ayushk4**
Refer #28

### CODE REVIEW

File: src/words/TokTok.jl [hunk 1]
```diff
@@ -101,51 +101,56 @@ function toktok_tokenize(instring::AbstractString)
...
     # handles FINAL_PERIOD_1 = r"(?<!\.)\.$"
     if length(ts.input) >= 2 && ts.input[end] == '.' && ts.input[end-1] != '.'
-        flush!(ts, ".")
         effective_end -= 1
-        return effective_end
+        return effective_end, ".", nothing
     end
 
     # handles FINAL_PERIOD_2 = r"(?<!\.)\.\s*(["'’»›”]) *$"
-    if ts.input[end] in "“”‘’›" || isspace(ts.input[end])
```
Comment (Ayushk4) [inline src/words/TokTok.jl:124]: Shouldn't this also have `\"` in it based on regex?

Comment (oxinabox) [reply src/words/TokTok.jl:124]: Yes

File: src/words/TokTok.jl [hunk 2]
```diff
@@ -54,14 +54,14 @@ const CURRENCY_SYM = (
         "\u20b4", "\u20b5", "\u20b6", "\u20b7", "\u20b8", "\u20b9", "\u20ba", "\ua838",
         "\ufdfc", "\ufe69", "\uff04", "\uffe0", "\uffe1", "\uffe5", "\uffe6", "\u09fb")
 
-const rules_atoms = Tuple(Iterators.flatten([
+const rules_atoms = collect.(Tuple(Iterators.flatten([
```
Comment (Ayushk4) [inline src/words/TokTok.jl:57]: The `String` error occurring in `atoms` was due to this. I tested it on the wiki corpus that @ComputerMaestro was getting error in, and now this works. Refer #29

Comment (Ayushk4) [reply src/words/TokTok.jl:57]: I have updated the testset for this as well.

**Comment by Ayushk4** [comment]
@oxinabox I believe this is done from my side, pls review.

**Comment by Ayushk4** [comment]
Here is one small difference b/w 1.1 and 1.0 that caused error in one of the newly added test cases. (Now fixed)

1.1 
```
julia> collect.(Iterators.flatten(["–—", ("'", "’")]))
4-element Array{Array{Char,N} where N,1}:
 '–'   
 '—'   
 ['\'']
 ['’'] 
```

1.0 
```
julia> collect.(Iterators.flatten(["–—", ("'", "’")]))
4-element Array{Array{Char,1},1}:
 ['–'] 
 ['—'] 
 ['\'']
 ['’'] 
```

**Comment by oxinabox** [comment]
LGTM, thanks

**Pull request merged**